subprojects/shipkit/src/test/groovy/org/shipkit/internal/gradle/versionupgrade/UpgradeDependencyPluginTest.groovy

```diff
@@ -111,11 +111,16 @@ class UpgradeDependencyPluginTest extends PluginSpecification {
         project.extensions.dependency = "org.shipkit:shipkit:0.1.2"
 
         when:
-        UpgradeDependencyExtension versionUpgrade = project.plugins.apply(UpgradeDependencyPlugin).upgradeDependencyExtension
+        project.plugins.apply(UpgradeDependencyPlugin)
         ReplaceVersionTask task = project.tasks.replaceVersion
 
         then:
-        task.versionUpgrade == versionUpgrade
+        with(task) {
```

Comment:
Can you verify if spock automatically adds "assert" in the statements below? Spock by default does not add implicit "assert" for code in blocks.